src/System.Data.SqlClient/src/System/Data/SqlClient/SNI/SNIPacket.cs

```diff
@@ -13,22 +13,12 @@ namespace System.Data.SqlClient.SNI
     /// </summary>
     internal class SNIPacket : IDisposable, IEquatable<SNIPacket>
     {
-        private byte[] _data;
-        private int _length;
-        private int _capacity;
+        private byte[] _dataBuffer;
```

Comment:
Can we keep these variable names the same as earlier to keep the extent of changes less? 
The name `_data` was good and its type pointed at the fact that it was a buffer. 
Similar comment for _length change to _dataLength